src/learn/zkcloud/snarkos.md

```diff
@@ -1 +1,26 @@
-# snarkOS
+# SnarkOS
+
+>SnarkOS is a decentralized operating system for zero-knowledge applications. This code forms the backbone of Aleo network, which verifies transactions and stores the encrypted state applications in a publicly-verifiable manner.
```

Comment:
Is it missing a word? "...stores the encrypted state OF applications in a publicly-verifiable manner...".